tsconfig.json

```diff
@@ -23,11 +23,9 @@
     "app.js"
   ],
   "typedocOptions": {
-    "mode": "library",
     "out": ".typedoc",
-    "entryPoint": "index",
+    "entryPoints": ["src/index.ts"],
     "excludePrivate": true,
-    "excludeNotExported": true,
     "exclude": ["**/index.ts"]
```

Comment:
This seems to exclude the entryPoint.
Does `entryPoints` trump `exclude`??

Reply:
:shrug: it seems to build the right thing with it excluded now.